exec_manager/dao/db_models.py

```diff
@@ -15,26 +15,22 @@
 
 """Defines all database specific ORM models"""
 
-from sqlalchemy import JSON, Boolean, Column, Integer, String
+
+from sqlalchemy import JSON, Column, String
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm.decl_api import DeclarativeMeta
 
 Base: DeclarativeMeta = declarative_base()
+metadata = Base.metadata
 
 
-class ExampleObjectA(Base):
-    """An example object stored in the DB"""
-
-    __tablename__ = "visas"
-    id = Column(Integer, primary_key=True)
-    name = Column(String, nullable=False)
-    some_json_details = Column(JSON, nullable=False)
-
+class DBJob(Base):
+    """An job object stored in the DB"""
 
-class ExampleObjectB(Base):
-    """Another example object stored in the DB"""
+    __tablename__ = "job"
 
-    __tablename__ = "table_b"
-    id = Column(Integer, primary_key=True)
-    name = Column(String, nullable=False)
-    active = Column(Boolean, nullable=False)
+    job_id = Column(String, primary_key=True)
```

Comment:
Und hier dann:
```suggestion
    job_id = Column(UUID(as_uuid=True), default=uuid.uuid4, primary_key=True)
```

So wird die UUID automatisch vergeben.